tests/scripts/validate_modified_files.sh

```diff
@@ -33,6 +34,9 @@ case "$1" in
   docs)
     validate "$DOCS_ERR"
   ;;
+  helm-docs)
+    validate "$HELM_DOCS_ERR"
```

Comment:
Before we merge, how about push a test commit with a helm change to verify that this validation catches the error?

Reply:
done. Let's see